### ISSUE TITLE
A question about the test result

### ISSUE THREAD

**Issue by khw321** [MaybeShewill-CV/lanenet-lane-detection#24]
@MaybeShewill-CV Hi, I have trained on CULane dataset, and this is a strange test result, have you ever met this?
And when two lanes are close, they are easy to become wider and be connected together, you can see it in the middle of the image.
![result_2](https://user-images.githubusercontent.com/24911307/42745145-b99c0ae2-8903-11e8-8ca9-539a5fd6de19.jpg)

**Comment by khw321** [comment]
@MaybeShewill-CV I got it, and the blue area in the right is caused by the cluster error.
I have another question about changing the result format to Tusimple's format,  My way is simply choosing the middle of the lane. **What is yours strategy, and can you give me some advice on using the Hnet?**

**Comment by MaybeShewill-CV** [comment]
@khw321 The HNet code has some bug for now and I will release it once it got fixed

**Comment by khw321** [comment]
@MaybeShewill-CV About the result format, what is your strategy to get the lane point?

**Comment by MaybeShewill-CV** [comment]
@khw321 The lane point is captured by the lanenet binary segmentation network. You can check the paper for details

**Comment by MaybeShewill-CV** [comment]
@khw321 You can use the binary segmentation result as mask to get the instance lane points on the image.

**Comment by khw321** [comment]
@MaybeShewill-CV OK, thanks!

**Comment by cardwing** [comment]
@khw321 , since you mentioned that you have trained your model in CULane dataset, I wonder how well your model performs (e.g., F1 measure in different road categories and in total). I just want to know the performance of the model so that I can decide whether I should use this code. Thanks a lot ! ^_^

**Comment by khw321** [comment]
@cardwing I only used the images which have more than 1 lane in CULane dataset for training, the F1 measure in total is 63.2

**Comment by PenghuahuaPeng** [comment]
@khw321 Hi，i have trained lanenet model in CULane dataset,but in first evl epoch i got a "nan" loss. I have tried to change learning rate,batch_size and so on,which has turned out to be no use.So,do you think
there is wrong with my traslated dataset?Can you show me your codes used for forming lanenet labels of culane dataset?Thank you very much.

**Comment by khw321** [comment]
@PenghuahuaPeng Sorry, I can't find my code now. You could check the label, the no lane label may cause error.

**Comment by PenghuahuaPeng** [comment]
Thanks a lot, i will have a try!

**Comment by PenghuahuaPeng** [comment]
@khw321 Sorry,excuse me,do you remember the final accuracy in culane dataset trained by these code?I have a 60-70 training accuracy and a 45 val accuracy after 60k epoch,now,it seems reach a balance,which is too slow.Thank you

**Comment by khw321** [comment]
@PenghuahuaPeng I only used the images which have more than 1 lane in CULane dataset for training, the F1 measure in total is 63.2

**Comment by abhigoku10** [comment]
@khw321  hi from the conversation above it is evident that u have trained a model on CU simple dataset is it possible for you to share the trained model

**Comment by TommyTang930** [comment]
@khw321 
您好，请问如何将CULane的training数据集转换成TuSimple数据集格式？麻烦指点一下

**Comment by khw321** [comment]
@TommyTang930 CULane和Tusimple都是在固定高度上取点，只不过CULane的label是每根车道线一行，包含每个点的w,h坐标，而Tusimple是将h提出来作为h_sample，单独一行，然后每根车道线的w坐标为一行，如果某个h上不存在车道线的点，那对应的w设为-2

**Comment by khw321** [comment]
@abhigoku10 Sorry, I can't find my trained model now.
